test/fixtures/output/network/access-point/private-link/egress-endpoint/create-help.golden

```diff
@@ -6,15 +6,11 @@ Usage:
 Examples:
 Create an AWS private link egress endpoint with high availability.
 
-  $ confluent network access-point private-link egress-endpoint create --cloud aws --gateway gw-123456 --service vpce-123456 --high-availability
-
-Create a named Azure private link egress endpoint.
-
-  $ confluent network access-point private-link egress-endpoint create my-egress-endpoint --cloud azure --gateway gw-123456 --service vpce-123456
+  $ confluent network access-point private-link egress-endpoint create --cloud aws --gateway gw-123456 --service com.amazonaws.vpce.us-west-2.vpce-svc-00000000000000000 --high-availability
 
 Flags:
-      --cloud string         REQUIRED: Specify the cloud provider as aws or azure.
-      --service string       REQUIRED: Name of an AWS VPC endpoint service or ID of an Azure Private Link service.
+      --cloud string         REQUIRED: Specify the cloud provider as aws.
```

Comment:
This is a bit awkward since there's only one option

Reply:
Yeah, it is a bit awkward. But we decided to keep it: https://confluent.slack.com/archives/C04PBPC4H1T/p1710292590197059?thread_ts=1710281192.969849&cid=C04PBPC4H1T